### PR TITLE
Create Oscar..ipynb

### PR DESCRIPTION
**Pull request by OscarAhumadaG**

### CODE REVIEW

File: Oscar.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import torch.optim as optim
+import numpy as np
+
+num_samples = 2000
+input_size = 10
+output_size = 1
+
+"""Create a random dataset for regression"""
+X = np.random.rand(num_samples, input_size).astype(np.float32)
+y = np.random.rand(num_samples, output_size).astype(np.float32)
```
Comment (Dour4do) [inline Oscar.py:12]: This is a comment

File: Oscar.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import torch
```
Comment (NayaVasques) [inline Oscar.py:1]: make a comment here!

File: Oscar.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+import torch.nn as nn
+import torch.optim as optim
+import numpy as np
```
Comment (Dopo0) [inline Oscar.py:4]: nice!!

File: Oscar.py [hunk 4]
```diff
@@ -1 +1,116 @@
...
+num_samples = 2000
+input_size = 10
+output_size = 1
+
+"""Create a random dataset for regression"""
+X = np.random.rand(num_samples, input_size).astype(np.float32)
+y = np.random.rand(num_samples, output_size).astype(np.float32)
+
+split_ratio = 0.75
+split_idx = int(num_samples * split_ratio)
```
Comment (OscarAhumadaG) [inline Oscar.py:15]: # This code needs to be reviewed by Kaveh

**Review by NayaVasques — APPROVED** [review]

**Review by Dopo0 — APPROVED** [review]
you corrected nice

**Review by hosseinijadpanah — APPROVED** [review]

**Review by Dour4do — APPROVED** [review]

**Pull request merged**